microprofile-rest-client/README.adoc

```diff
@@ -11,6 +11,7 @@ The `microprofile-rest-client` quickstart demonstrates the use of the MicroProfi
 :standalone-server-type: default
 :archiveType: jar
 :archiveName: {artifactId}
+:jbossHomeName: JBOSS_HOME
```

Comment:
Please confirm this is really needed, such attribute value is defined by included /shared-doc/attributes.adoc, and used in other shared-docs includes